### ISSUE TITLE
language dependent date/time formating

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#602]
I have been looking at how the 'modified on' date is formatted across a number of different language wikis and have noticed that each language potentially has different formating

The most dramatic I have found are (for the languages I have worked with) Hebrew (he) and Japanese (ja)

To that end my branch date_format attempts to acknowledge this

I have introduce a new 'command' within the language gfs files `time_formats` that gives the formats associated with that language

My experimentation has only extended to six languages (en, fr, it, de, he and ja) which I have manually amended.
I am happy to produce a more automated system (probably in python) to generate the appropriate formatting across the other languages if this seems the right approach

Basically, from the MessagesXX.php file there can be
$dateFormats
$datePreferences
$defaultDateFormat
if they don't exist the default is from EN

For EN I have chosen that the default is dmy (it can be either dmy or mdy depending on other things). This seems to display the modified on date in the same way as mediwiki
